src/main/java/com/google/sps/servlets/EventServlet.java

```diff
@@ -116,10 +114,14 @@ private void createEntity(HttpServletRequest request, Date endDate, Date startDa
     String yesCOVIDSafe = request.getParameter(Constants.COVID_SAFE_PARAM);
     String privacy = request.getParameter(Constants.PRIVACY_PARAM);
     String attendeesString = request.getParameter(Constants.ATTENDEES_PARAM);
-    List<String> attendees = Arrays.asList(attendeesString.split("\\s*,\\s*"));
+    List<String> attendeesList = Arrays.asList(attendeesString.split("\\s*,\\s*"));
+    ArrayList<String> attendees = new ArrayList<String>(attendeesList);
```

Comment:
What was the reason for having this as a List and then creating it again as an ArrayList? Is it because only an ArrayList has the functionality you needed to check and add the creator ID?

Reply:
Yes, I couldn't check contains if it isn't an arraylist and this was the only way I could seem to get it to work. It is inefficient tho so I would be open to suggestion if you had any